rdkcompositor.cpp

```diff
@@ -593,4 +660,27 @@ namespace RdkShell
     {
         return mReceivedKeyPress;
     }
+
+    void RdkCompositor::getVirtualResolution(uint32_t &virtualWidth, uint32_t &virtualHeight)
+    {
+        virtualWidth = mVirtualWidth;
+        virtualHeight = mVirtualHeight;
+    }
+
+    void RdkCompositor::setVirtualResolution(uint32_t virtualWidth, uint32_t virtualHeight)
+    {
+        mVirtualWidth = virtualWidth;
+        mVirtualHeight = virtualHeight;
```

Comment:
Let's update to `mVirtualWidth = (virtualWidth > 0) ? virtualWidth : mWidth;` and  `mVirtualHeight = (virtualHeight > 0) ? virtualHeight : mHeight;`